canonical-webteam/docker-bundler
# Add variable to speed up building of nokogiri

QA
--

Build this branch, as a local image called `bundler`:

``` bash
docker build --tag bundler .
```

Pull down and run vanillaframework.io:

``` bash
git clone https://github.com/canonical-websites/vanillaframework.io
cd vanillaframework.io
./run
```

Note how long it stops on the line `Installing nokogiri 1.7.2 with native extensions`. Probably more than 10 seconds.

Now, edit the `./run` script and replace "`canonicalwebteam/bundler:v0.1.1`" with "`bundler`" - the version built from this branch.

Then run:

``` bash
./run clean
./run
```

This time, when it gets to `Installing nokogiri 1.7.2 with native extensions`, it will still pause a bit, but it should complete an order of magnitude faster.

Check the site still runs okay.
